<launch>
  <rosparam command="load" file="$(find mur_sensors)/config/sensor_config.yaml" />
  <group ns="sensors">
    <node pkg="mur_sensors" name="sensors_receive_server" type="ros_sensor_udp_server_dyn.py" output="screen"/>
    <node pkg="mur_sensors" name="imu_fusion" type="ros_imu_sensor_fusion_dyn.py" output="screen"/>
    <node pkg="mur_sensors" name="sensors_auto_network" type="ros_sensor_udp_auto_network.py" output="screen"/>
  </group>
</launch>